spec/acceptance/solr5_install_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper_acceptance'
+
+describe 'solr5 class' do
+
+    describe 'running puppet code' do
+
+        it 'should work with no errors' do
```

Comment:
it `should install solr as a service and start it` 

and then add the necessary expectations :)